tests/phpunit/tests/block-template-utils.php

```diff
@@ -187,6 +187,28 @@ function test_inject_theme_attribute_in_block_template_content() {
 		$this->assertEquals( $content_with_no_template_part, $template_content );
 	}
 
+	function test_remove_theme_attribute_in_block_template_content() {
+		$content_with_existing_theme_attribute = '<!-- wp:template-part {"slug":"header","theme":"tt1-blocks","align":"full","tagName":"header","className":"site-header"} /-->';
+		$template_content                      = _remove_theme_attribute_in_block_template_content( $content_with_existing_theme_attribute );
+		$expected                              = '<!-- wp:template-part {"slug":"header","align":"full","tagName":"header","className":"site-header"} /-->';
+		$this->assertEquals( $expected, $template_content );
```

Comment:
```suggestion
		$this->assertSame( $expected, $template_content );
```

:1234: Applies to each item for this test to be type strict.